Discovery/DetectTorrentUse

```diff
@@ -0,0 +1,4 @@
+//Custom detection to find use of torrenting software or browsing related to torrents
+NetworkCommunicationEvents
+| where RemoteUrl contains "torrent" or RemoteUrl contains "vuze" or RemoteUrl contains "azureus" or RemoteUrl endswith ".tor" or InitiatingProcessFileName contains "torrent" or InitiatingProcessFileName contains "vuze" or InitiatingProcessFileName contains "azureus" 
```

Comment:
For performance, is it possible to replace `contains` operator with `has` in this context?
`has` operator looks for "full term", meaning `text has "be"` will match `let it be.` but not `because`.